editor/gui/editor_run_bar.cpp

```diff
@@ -50,7 +51,34 @@ void EditorRunBar::_notification(int p_what) {
 			_reset_play_buttons();
 		} break;
 
+		case NOTIFICATION_READY: {
+			if (Engine::get_singleton()->is_safe_mode_hint()) {
+				safe_mode_show_dialog();
+			}
+		} break;
+
 		case NOTIFICATION_THEME_CHANGED: {
+			if (Engine::get_singleton()->is_safe_mode_hint()) {
+				main_panel->add_theme_style_override("panel", get_theme_stylebox(SNAME("LaunchPadSafeMode"), EditorStringName(EditorStyles)));
+				safe_mode_panel->add_theme_style_override("panel", get_theme_stylebox(SNAME("SafeModeButton"), EditorStringName(EditorStyles)));
```

Comment:
```suggestion
				main_panel->add_theme_style_override(SceneStringName(panel), get_theme_stylebox(SNAME("LaunchPadSafeMode"), EditorStringName(EditorStyles)));
				safe_mode_panel->add_theme_style_override(SceneStringName(panel), get_theme_stylebox(SNAME("SafeModeButton"), EditorStringName(EditorStyles)));
```
New constant

Reply:
Fixed, thanks :+1: (had to create a new StringName for this one)

Reply:
What? It's right there if you rebase your branch :)

Reply:
Oh yeah, my bad, I **really** need to make a habit of always rebasing before force-pushing :sweat_smile: